### PR TITLE
Add notes for Surah Asar and Surah Humaza

### PR DESCRIPTION
**Pull request by mushtaqak**
This PR adds notes for 103: Surah Asar and 104: Surah Humaza

### CODE REVIEW

File: notes/roman-urdu/103_Surah_Asar.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Number 103: Surah Asar
+Ayahs: 3
+Rukus: 1
+Related Surah: [104. Surah Humaza](notes/roman-urdu/104_Surah_Humaza.md)
+
+### Zamane ki qasam Insan bare khasare m h
```
Comment (miankhalid) [inline notes/roman-urdu/103_Surah_Asar.md]: mein hai

File: notes/roman-urdu/103_Surah_Asar.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+### Sawai wo log jo in 4 kaam poore karta ho
```
Comment (miankhalid) [inline notes/roman-urdu/103_Surah_Asar.md]: Siwaye wo log jo in 4 kaamon ko poora kartay hon

File: notes/roman-urdu/103_Surah_Asar.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+1. Jinho ne imaan lana.
+2. Naik amal
+3. Haq ki naseehat.
+4. Sabar ki talqain.
+5. Allah par uske rasool malaik aakhrat par imaan la kar phr Allah k batae huey un baato par amal kre. na sirf khud balqai wo haq/islam ki baat dusro tak pohchaye ... jb yeh kam kre ga to museebatein aaye gi. unka sabar se kaam le.
```
Comment (miankhalid) [inline notes/roman-urdu/103_Surah_Asar.md]: Nits:
- 4 kaam alag or final description alag alag honay chahiyein.
- Allah par, uskay rasool par, malaika par aur akhirat par imaan la kar phir Allah k batai hoi baato par amal kre. Na sirf khud balqai wo haq/islam ki baat dusro tak pohchaye ... jb yeh kam kre ga to museebatein aaye gi. Tab ja k sabar se kaam le.

File: notes/roman-urdu/103_Surah_Asar.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+Rukus: 1
+Related Surah: [104. Surah Humaza](notes/roman-urdu/104_Surah_Humaza.md)
+
+### Zamane ki qasam Insan bare khasare m h
+
+### Sawai wo log jo in 4 kaam poore karta ho
+1. Jinho ne imaan lana.
+2. Naik amal
+3. Haq ki naseehat.
+4. Sabar ki talqain.
```
Comment (miankhalid) [inline notes/roman-urdu/103_Surah_Asar.md]: 1 - Jo imaan laaye.
2 - Naik amal karay.
3 - Haq ke naseehat karay.
4 - Or sabar ke talqeen karay.

File: notes/roman-urdu/104_Surah_Humaza.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+Number 104: Surah Humaza
+Ayahs: 9
+Rukus: 1
+Related Surah: [103. Surah Asar](notes/roman-urdu/103_Surah_Asar.md)   
+
+### Info
+ Jo surah Asar k rasty par ni chalty yeh surah unka qurdaar dikha rhi h
```
Comment (miankhalid) [inline notes/roman-urdu/104_Surah_Humaza.md]: kirdaar

File: notes/roman-urdu/104_Surah_Humaza.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+### Kharabi uske liye jo
+1. Geebat kerta ho
+2. Tanabazi karta ho
```
Comment (miankhalid) [inline notes/roman-urdu/104_Surah_Humaza.md]: Taanay baazi

File: notes/roman-urdu/104_Surah_Humaza.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+3. Maal jama krne wala ho aor uski ginti krta ho
+4. smjhta h maal usko hamesha zinda rakhy gq
```
Comment (miankhalid) [inline notes/roman-urdu/104_Surah_Humaza.md]: `Smjhta` h k maal usko hamesha zinda rakhay `ga`

File: notes/roman-urdu/104_Surah_Humaza.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+### Hutama
+1. Us skhs ko Hutama m daala jaye ga jo.
```
Comment (miankhalid) [inline notes/roman-urdu/104_Surah_Humaza.md]: Us shakhs ko Hutama m daala jaye ga

**Review by miankhalid — CHANGES_REQUESTED** [review]
Some nits and a general suggestion to either use full stops everywhere or not use them at all in case of bullet points.

**Comment by mushtaqak** [comment]
@miankhalid addressed your feedback. Kindly review again :)

**Review by miankhalid — APPROVED** [review]
Jazak Allah

**Pull request merged**